src/app/zap-templates/zcl/data-model/chip/chip-types.xml

```diff
@@ -57,39 +55,57 @@ limitations under the License.
+    <type id="0xEE" description="Action ID"                name="action_id"         size="1"  discrete="true" />
     <type id="0xEF" description="Transaction ID"           name="trans_id"          size="4"  discrete="true" />
     <type id="0xF0" description="Node ID"                  name="node_id"           size="8"  discrete="true" />
-    <type id="0xF1" description="Vendor ID"                name="vendor_id"         size="2"  discrete="true" />
-    <type id="0xF2" description="Device Type ID"           name="devtype_id"        size="4"  discrete="true" />
-    <type id="0xF3" description="Fabric ID"                name="fabric_id"         size="8"  discrete="true" />
-    <type id="0xF4" description="Group ID"                 name="group_id"          size="2"  discrete="true" />
-    <type id="0xF5" description="Status Code"              name="status"            size="1"  discrete="true" />
-    <type id="0xF6" description="Data Version"             name="data_ver"          size="4"  discrete="true" />
-    <type id="0xF7" description="Event Number"             name="event_no"          size="8"  discrete="true" />
-    <type id="0xF8" description="Endpoint Number"          name="endpoint_no"       size="2"  discrete="true" />
-    <type id="0xF9" description="Fabric Index"             name="fabric_idx"        size="1"  discrete="true" />
-    <type id="0xFA" description="IP Address"               name="ipadr"                       composite="true"/>
-    <type id="0xFB" description="IPv4 Address"             name="ipv4adr"           size="4"  composite="true"/>
-    <type id="0xFC" description="IPv6 Address"             name="ipv6adr"           size="16" composite="true"/>
-    <type id="0xFD" description="IPv6 Prefix"              name="ipv6pre"                     composite="true"/>
-    <type id="0xFE" description="Hardware Address"         name="hwadr"                       composite="true"/>
+
+    <type id="0xC6" description="Entry Index"              name="entry_idx"         size="2"  discrete="true" />
+    <type id="0xC7" description="Data Version"             name="data_ver"          size="4"  discrete="true" />
+    <type id="0xC8" description="Event Number"             name="event_no"          size="8"  discrete="true" />
+
+    <type id="0xF2" description="IP Address"               name="ipadr"                       composite="true"/>
+    <type id="0xF3" description="IPv4 Address"             name="ipv4adr"           size="4"  composite="true"/>
+    <type id="0xF4" description="IPv6 Address"             name="ipv6adr"           size="16" composite="true"/>
+    <type id="0xF5" description="IPv6 Prefix"              name="ipv6pre"                     composite="true"/>
+    <type id="0xF6" description="Hardware Address"         name="hwadr"                       composite="true"/>
+
+    <type id="0xC9" description="Semantic Tag"             name="semtag"            size="4"  discrete="true" />
```

Comment:
Could you add the new types to https://github.com/project-chip/connectedhomeip/blob/master/scripts/py_matter_idl/matter_idl/generators/types.py#L165 as well?

Reply:
done.